_posts/2021-03-20-march-meetup.md

```diff
@@ -7,7 +7,7 @@ description: "In collaboration with PythonPune"
 categories: [ meetup, talks, workshop ]
 image:
 featured: true
-hidden: false
+hidden: true
```

Comment:
Won't this hide the post itself?

Reply:
No, it hides only from the regular list.

So if a post is featured we can hide it from the most regular list of posts.